src/main/java/org/broadinstitute/hellbender/utils/hdf5/HDF5PoN.java

```diff
@@ -16,9 +16,20 @@
 /**
  * HDF5 File backed Panel of Normals data structure.
  *
- * Several attributes are stored transposed, in order to dodge a very slow write time in HDF5.
- *  Normalized Counts
- *  Log-Normalized Counts
+ * Several attributes are stored transposed (in other words, the rows are columns and vice versa).
```

Comment:
I don't think "the rows are columns and vice versa" is what you meant to say here. . .